src/client.ts

```diff
@@ -117,6 +125,18 @@ export default class Client implements ClientInterface {
         formatter: configuration.get("formatter"),
       },
       middleware: {
+        provideCodeLenses: async (document, token, next) => {
+          if (this.client) {
+            const response = await next(document, token);
+
+            if (response) {
+              this.testController.createTestItems(response as CodeLens[]);
```

Comment:
Is this assuming that all code lenses will be testing related? What happens if the LSP server decides to add other kinds of code lenses? Will this code break?